subcommand/version/command.go

```diff
@@ -12,6 +12,7 @@ import (
 type Command struct {
 	UI      cli.Ui
 	Version string
+	FIPS    string
```

Comment:
Are we intending to output this in the `consul-ecs version` command? I don't think that it will actually get reported since we only report `c.Version` in the CLI output: https://github.com/hashicorp/consul-ecs/blob/7b44fa3510d07f2874bff13b0b4df3cf8aae6703/subcommand/version/command.go#L19

Reply:
Removed - thank you for catching. For consistency wrt to smoke tests and user scripts, and given that this is OSS so the FIPS build bits are available for inspection, we aren't changing the version format to verbose multi-field/multi-line in k8s, dataplane, nor ECS.

Reply:
:ok_hand: Thanks for the explanation.